site/_docs/history.md

```diff
@@ -27,24 +27,120 @@ For a full list of releases, see
+{: #v1-35-0}
+
+This release comes 4 months after [1.34.0](#v1-34-0),
+contains contributions from 36 contributors, and resolves 139 issues.
+
+Among other new features, we have added extensive function support. For example, we added roughly 14 new functions in BigQuery library for handling array, dates, math, and string.
+Also, we added 27 new functions in Spark library for handling array, map and about 11 new functions for other database products.
+
+Among other improvements and fixes, it is worth highlighting the following improvements:
+
+* Some improvements in calcite core.
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5703">CALCITE-5703</a>]
+    Reduce amount of generated runtime code
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5479">CALCITE-5479</a>]
+    FamilyOperandTypeChecker is not readily composable in sequences
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5425">CALCITE-5425</a>]
+    Should not pushdown Filter through Aggregate without group keys
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5506">CALCITE-5506</a>]
+    RelToSqlConverter should retain the aggregation logic when Project without RexInputRef on the Aggregate
+
+* Some improvements in simplify an expression.
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5769">CALCITE-5769</a>]
+    Optimizing 'CAST(e AS t) IS NOT NULL' to 'e IS NOT NULL'
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5780">CALCITE-5780</a>]
+    Simplify '1 > x OR 1 <= x OR x IS NULL' to TRUE
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5798">CALCITE-5798</a>]
+    Improve simplification of '(x < y) IS NOT TRUE' when x and y are not nullable
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5759">CALCITE-5759</a>]
+    'SEARCH(1, Sarg[IS NOT NULL])' should be simplified to 'TRUE'
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5639">CALCITE-5639</a>]
+    RexSimplify should remove IS NOT NULL check when LIKE comparison is present
+
+Contributors to this release:
+Aitozi,
+Akshay Dayal,
+Benchao Li,
+Charles Givre,
+Clay Johnson,
+Guillaume Massé,
+ILuffZhe,
+Itiel Sadeh,
+Julian Hyde,
+Lei Zhang,
+Leonid Chistov,
+Mihai Budiu,
+Oliver Lee,
+Ran Tao,
+Roman Kondakov,
+Runkang He,
+Sergey Nuyanzin,
+Stamatis Zampetakis,
+TJ Banghart,
+Tanner Clary,
+Tim Nieradzik,
+Will Noble,
+adamk,
+dssysolyatin,
+guofeng.my,
+ian.bertolacci,
+jhugomoore,
+macroguo,
+nobigo (release manager),
+rubenada,
+xiejiajun,
+yongen.ly,
+zhujiang.97,
+zoudan,
+zstan,
+何润康.
+
 #### Breaking Changes
 {: #breaking-1-35-0}
 
-The way of Locale parsing changed within [<a href="https://issues.apache.org/jira/browse/CALCITE-5567">CALCITE-5567</a>]
-Now locale's language tag should match IETF BCP 47 language tag or be empty.
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5823">CALCITE-5823</a>]
+  Remove commons-collections dependency from innodb
```

Comment:
Why is this a breaking change? I though that this change is transparent to users of calcite.

Reply:
If the user uses this dependency in their project which was introduced by Calcite, We need to tell it.